tools/tests/organizations.js

```diff
@@ -36,54 +36,3 @@ selftest.define("organizations - logged out", function () {
   run.stop();
 
 });
-
-// For now, this test only runs from checkout with a universe file
-// pointing to a testing meteor-accounts server (e.g. one deployed with
-// Meteor.settings.testing = true).
-selftest.define("organizations", ["net", "slow", "checkout"], function () {
```

Comment:
Honestly, I didn't actually try to run it — I just took it at its word that it required special setup to run and assumed it had bitrotted over the years, as many of our `--slow` tests have.

Running it now (with `curmeteor self-test --slow '^organizations$'`) it does actually appear to pass.  (Note: it does so by actually creating a real org with name starting with `selftestorg` in the production Meteor Developer Accounts server, and it doesn't clean them up afterwards; there is a cronjob in the server that cleans up these orgs daily.)

Would you like me to keep this test and just delete the part of the comment that claims it doesn't work with the production server? I have to say I personally don't love tests that mutate production servers, but I guess it works.